packages/swingset/src/resolvers/component.ts

```diff
@@ -67,6 +68,7 @@ export async function resolveComponents({
         relativePath,
         slug: normalizedNestedPath,
         title,
+        parsedPath: parseComponentPath(componentPath)
```

Comment:
We don't need to add `parsedPath` here, as nested documents live outside of the primary navigation tree. This should also fix the case where you were seeing an error locally